src/components/Global/ImageGeneration/LinkPreview.tsx

```diff
@@ -133,7 +133,7 @@ export function LinkPreviewImg({
                             color: 'black',
                         }}
                     >
-                        {utils.formatTokenAmount(parseFloat(amount), 2)} {tokenSymbol}
+                        {utils.formatAmount(amount)} {tokenSymbol}
```

Comment:
@Hugo0 i tired to update the codebase to use the updated function in link preview component too, but unfortunately the fix didn't work as expected, it looks like a caching issue to me, i'm looking into this more deeply

Reply:
@kushagrasarathe any update here?

Reply:
@Hugo0 nothing sofar, tried changing the font size in a test commit, results are still same, can you try clearing the cache from vercel dashboard once?
<img width="627" alt="Screenshot 2024-12-23 at 10 29 05 PM" src="https://github.com/user-attachments/assets/29fd6b3f-920b-4f0d-b1cd-da974ac64d2e" />